### DISCUSSION TITLE
List of columns for change hash

### DISCUSSION THREAD

**Discussion by marcinsiewiera** [ScalefreeCOM/datavault4dbt#108] [Q&A] [answered]
Hi All,

Is there a way to define columns for change hash in stage macro by specifying columns to be excluded rather than listing all columns 1 by 1?
Instead of using 

    hd_lineitem_n_s:
        is_hashdiff: true
        columns:
            - col1
            - col2
            - col3
            - ....
            - col50
            - ...

exclude business key 

        columns:
          - include: '*' 
            exclude:
              - businessKey
              - hashKey
              - changeHash
              - loaddate
              - recordsource
              - ...

Regards,
Marcin

**Comment by tkirschke** [comment, marked as answer]
Hi @marcinsiewiera and thanks for reaching out!

This is a great feature suggestion and I would love to have it in the package!
I will definitely add it to our list of feature requests, but that might take a while until it is being worked on. 

So I really want to encourage you to try modifying the stage macro accordingly and do a Pull Request!

Best regards, 
Tim
